src/components/DeviceSelectionDialog/DeviceSelectionDialog.tsx

```diff
@@ -72,12 +73,14 @@ export default function DeviceSelectionDialog({ open, onClose }: { open: boolean
           <AudioOutputList />
         </div>
         <Divider />
```

Comment:
I think this <Divider> needs to go in the <Hidden>, so we don't end up with a second line like this:


![image](https://user-images.githubusercontent.com/12685223/174345274-4e5405fb-a699-48d4-aeba-7c95621bd3b4.png)
![image](https://user-images.githubusercontent.com/12685223/174345274-4e5405fb-a699-48d4-aeba-7c95621bd3b4.png)